`runtime error: index out of range [0] with length 0` - Teleport Desktop Access crashed

### Description

**What happened**:

While testing I had Teleport Desktop Access service crash a few times. The specific panic was `panic: runtime error: index out of range [0] with length 0`

```
2022-03-02T04:45:02Z INFO [WINDOWS_D] TDP input streaming finished client-ip:35.88.204.73 desktop-addr:10.0.133.32:3389 desktop-name:EC2AMAZ-1EEIF4O-teleport9-com rdp-addr:10.0.133.32:3389 runtime/panic.go:1047
panic: runtime error: index out of range [0] with length 0

goroutine 858 [running]:
github.com/gravitational/teleport/lib/srv/desktop/rdp/rdpclient.(*Client).start.func2.5(0xc000c93450, {0x7f89800, 0x575b2c0, 0xc000112100})
	/go/src/github.com/gravitational/teleport/lib/srv/desktop/rdp/rdpclient/client.go:367 +0x91
github.com/gravitational/teleport/lib/srv/desktop/rdp/rdpclient.(*Client).start.func2()
	/go/src/github.com/gravitational/teleport/lib/srv/desktop/rdp/rdpclient/client.go:368 +0x434
created by github.com/gravitational/teleport/lib/srv/desktop/rdp/rdpclient.(*Client).start
	/go/src/github.com/gravitational/teleport/lib/srv/desktop/rdp/rdpclient/client.go:261 +0xcf
```

**What you expected to happen**:
not crash


### Server Details
- Teleport version (run `teleport version`): 9.0.0-beta.1
- Server OS (e.g. from `/etc/os-release`):
- Where are you running Teleport? (e.g. AWS, GCP, Dedicated Hardware):
- Additional details:

--- a/lib/srv/desktop/rdp/rdpclient/client.go
+++ b/lib/srv/desktop/rdp/rdpclient/client.go
@@ -362,13 +362,17 @@ func (c *Client) start() {
 					return
 				}
 			case tdp.ClipboardData:
-				if err := cgoError(C.update_clipboard(
-					c.rustClient,
-					(*C.uint8_t)(unsafe.Pointer(&m[0])),
-					C.uint32_t(len(m)),
-				)); err != nil {
-					c.cfg.Log.Warningf("Failed forwarding RDP clipboard data: %v", err)
-					return
+				if len(m) > 0 {
+					if err := cgoError(C.update_clipboard(
+						c.rustClient,
+						(*C.uint8_t)(unsafe.Pointer(&m[0])),
+						C.uint32_t(len(m)),
+					)); err != nil {
+						c.cfg.Log.Warningf("Failed forwarding RDP clipboard data: %v", err)
+						return
+					}
+				} else {
+					c.cfg.Log.Warning("Recieved an empty clipboard message")
 				}
 			default:
 				c.cfg.Log.Warningf("Skipping unimplemented TDP message type %T", msg)

